pkg/core/localize/locales/en/cmd/connectors.toml

```diff
@@ -143,6 +143,36 @@ rhoas connector create --file=myconnector.json
 cat myconnector.json | rhoas connector create
 '''
 
+[connector.stop.cmd.shortDescription]
+one = 'Stop a Connectors instance'
+
+[connector.stop.cmd.longDescription]
+one = 'Stop a Connectors instance, pass an id or use the instance in the current context'
+
+[connector.stop.cmd.example]
+one = '''
+# Stop a Connectors instance
+rhoas connector stop
+
+# Stop a Connectors instance
+rhoas connector stop --id=IJD76DUH675234
+'''
+
+[connector.start.cmd.shortDescription]
+one = 'Start a Connectors instance'
+
+[connector.start.cmd.longDescription]
+one = 'Start a Connectors instance, pass an id or use the instance in the current context'
+
+[connector.start.cmd.example]
+one = '''
+# Start a Connectors instance
```

Comment:
@MelissaFlinn told me once to use `connector instance` as proper term. 
No plurals etc.